### DISCUSSION TITLE
How to made the tag pictures available?

### DISCUSSION THREAD

**Discussion by dream7180** [jcorporation/myMPD#963] [Q&A] [answered]
I create two folders "Album" and "AlbumArtist" under /var/lib/mympd/pics/, and copy some pic to them for testing.
![image](https://user-images.githubusercontent.com/18242221/215314818-9099176d-90fb-4a25-80c7-d3300b749de9.png)
However mympd was unable to load these pics.
And in settings, the Alubmart directory is always unavailable.
![image](https://user-images.githubusercontent.com/18242221/215314904-9fd6dc43-1cfb-4028-ab0e-54c6e787f654.png)

**Comment by jcorporation** [comment, marked as answer]
The tag picture semantic does not work for albums. To display albumart set the music directory in the connection dialog. For all other tags your attempt should work.

**Comment by dream7180** [comment]
Should I include /var/lib/mympd/pics or /var/lib/mympd/pics/AlbumArtist or /var/lib/mympd/pics/Artist in the the connection Music directory as below? But none of them works. Should I edit /etc/mpd.conf? Thanks.
![image](https://user-images.githubusercontent.com/18242221/215338468-e346c733-5809-4d1d-b1e4-539abe79584b.png)

**Reply by jcorporation** [reply]
No, no and no. Put there the path of your mpd music directory. 

Read: https://jcorporation.github.io/myMPD/references/pictures

**Comment by dream7180** [comment]
I use auto-detect before, it work fine for me although the warning exists. I use specify the music path now and the warning disappear. But can not return back to auto-detect any more.
However, the Artist or Albumartist pics still unavailable even I have done as instruction.
![image](https://user-images.githubusercontent.com/18242221/215418610-eca835bd-5a40-49ae-82d1-6207328e0b9d.png)

![image](https://user-images.githubusercontent.com/18242221/215418971-7c7c4a52-1e34-478e-8c61-ad0d8bb058c1.png)

**Reply by jcorporation** [reply]
Autodetect works only with a socket connection, not with "localhost" as mpd host. This is a MPD protocol restriction. In the playback view there is only albumart displayed. Artist images are only shown in the Browse -> Artist view.

**Reply by dream7180** [reply]
Get it now, thanks. Is it possible to make playback support the non-embedded albumarts named like "%albumartist% - %album%.jpg" that all put in one directory? Most of my cover pics are placed in such way that used by foobar2000 in my Windows PC. It will be great if mympd can read these "album tag pics".

**Reply by jcorporation** [reply]
I do not like this approach. Managing the pictures in the album folders is more flexible in my opinion.
